Makefile

```diff
@@ -0,0 +1,26 @@
+BUILD_DIR := build
+MKDIR_P = mkdir -p
+RM = rm -rf
+
+default: build-all
+
+load-deps:
+	${MKDIR_P} ${CURDIR}/${BUILD_DIR}
+	./pre-build.sh
+	touch $@
+
+%:
+	$(MAKE) load-deps clean-up
+	./mkimage ${CURDIR}/${BUILD_DIR}/$@.tar $@
+
+build-all: load-deps clean-up build
```

Comment:
This is a bit weird and not related to the Makefile I think.
```
Building base in /tmp/tmp.prz81D07DZ
E: No such script: /tmp/tmp.q1ocKi5sxE/scripts/build-all
```

Could you just run `./buildall` on your machine ?